include/swift/AST/DiagnosticsSema.def

```diff
@@ -3701,6 +3701,10 @@ WARNING(pbd_never_used_stmtcond, none,
         "value %0 was defined but never used; consider replacing "
         "with boolean test",
         (Identifier))
+WARNING(unused_setter_newvalue, none,
+        "setter argument %0 was never used, but the getter was accessed; "
+        "consider using %0 or removing %1",
```

Comment:
I think the "removing" phrasing is confusing. How about something like "setter argument %0 was never used; did you mean to use it instead of accessing the property's current value"?

Ideally, we'd put the warning on one of the uses of the getter, and maybe even have a fix-it if it's just one use, but that's probably more work than you want to do right now.